CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- update honeysql to 1.0.461 to distribute latest version to all consuming projects
```

Comment:
Can you update this to mention that this is to resolve security issues?

also (nit) should have a newline between this and the next section.